delfin/drivers/hpe/hpe_msa/hpe_msastor.py

```diff
@@ -0,0 +1,48 @@
+from delfin.drivers import driver
+from delfin.drivers.hpe.hpe_msa import ssh_handler
+from delfin.drivers.hpe.hpe_msa.ssh_handler import SSHHandler
+
+
+class HpeMsaStorDriver(driver.StorageDriver):
+
+    def __init__(self, **kwargs):
+        super().__init__(**kwargs)
+        self.ssh_hanlder = ssh_handler.SSHHandler(**kwargs)
+
+    def reset_connection(self, context, **kwargs):
+        self.ssh_hanlder.login()
+
+    def get_storage(self, context):
+        return self.ssh_hanlder.get_storage(self.storage_id)
```

Comment:
If almost all the function of ssh_handler need the parameter `storage_id`, it can be the member variable of ssh_handler